license-details/README.md

```diff
@@ -0,0 +1,147 @@
+# Introduction
+
+The P4 compiler `p4c` is a leading project of the P4 Consortium, and
+used by several companies as the front end of P4 compilers for a
+variety of hardware targets.  Most of these companies develop
+significant proprietary back end code, and the resulting binary
+executables are released, with no copyleft obligation for those
+companies to release their back end source code.
+
+This is allowed, because the `p4c` front source files are licensed
+under the Apache 2.0 license.  This license was chosen for multiple
+reasons, but in part to enable companies to do this.  It is the P4
+Consortium's intent to continue enabling this.
+
+
+# Linux Foundation review of P4.org software licenses in published code
+
+In early 2024 P4.org became part of The Linux Foundation.  In November
+2024 the Linux Foundation appointed someone to do a quick review of
```

Comment:
I do not know how formal this document is supposed to be but "someone" sounds a little too non-specific.

Reply:
I don't exactly feel comfortable putting their name in here without asking them.  My intent is that this file will be around for as long as this repository will exist.  Any alternate suggestions?

Reply:
I made this change in commit 8.